### ISSUE TITLE
TypeError: Cannot read property 'tap' of undefined in DEV

### ISSUE THREAD

**Issue by danielzzz** [preactjs/next-plugin-preact#19]
hi, I am getting a webpack error when trying to run with this plugin in dev.
 Production seems to be working fine.
Do you have any idea what might be causing it?

package.json
```
"webpack": "latest",
"next-plugin-preact": "^3.0.3",
"preact": "^10.5.10",
"preact-render-to-string": "^5.1.12",
"react": "npm:@preact/compat",
"react-dom": "npm:@preact/compat",
 "react-ssr-prepass": "npm:preact-ssr-prepass",
```

it happens only when using:
```
const withPreact = require('next-plugin-preact');
```

I seems that the aliases in package.json make it work even so, as the build size keeps reduced even if I disable it.

cheers, dan

**Comment by Drew-Garratt** [comment]
Seeing the same behaviour as above

**Comment by iam4x** [comment]
Yes, next v10.0.6 breaks the plugin. For now you can fix your next dependency in your package.json =>

```
"next": "10.0.5",
```

**Comment by ronanlevesque** [comment]
hey @iam4x 👋 

I tried downgrading to `10.0.5`, but I still get the `Cannot read property 'tap' of undefined` error

**Comment by iam4x** [comment]
Hey, can you run `npm ls next` and `npm ls webpack` and make sure you are using `next@10.0.5` and `webpack@^5.21.2` ?

![image](https://user-images.githubusercontent.com/893837/107349599-b190ea80-6ac8-11eb-9472-d5653f911c3a.png)

![image](https://user-images.githubusercontent.com/893837/107349632-bb1a5280-6ac8-11eb-904b-c3cf5b55d6cf.png)

With these two versions set, and clean node_modules folder I have no issues:

![image](https://user-images.githubusercontent.com/893837/107349696-d1281300-6ac8-11eb-87f1-f763964ba288.png)

**Comment by ronanlevesque** [comment]
`npm ls next` returns `(empty)` for some reason, but my `yarn.lock` correcty shows `next@10.0.5`.

`npm ls webpack`  returns `5.21.2` indeed

And I still get the error after a clean install 😞

**Comment by geisterfurz007** [comment]
This appears to be the same as https://github.com/JoviDeCroock/prefresh/issues/276

**Comment by JoviDeCroock** [comment]
Hey @danielzzz is this issue still present with the newer versions of prefresh & this plugin?
